### PR TITLE
tests: use pollinate to seed the rng

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

**Comment by chipaca** [comment]
@mvo5 AIUI pollinate is about giving it an initial bump of entropy to the pool; rng-tools is about keeping the pool full.

File: tests/lib/prepare.sh [hunk 1]
```diff
@@ -164,6 +164,11 @@ EOF
     fi
 
     if [[ "$SPREAD_SYSTEM" == debian-* || "$SPREAD_SYSTEM" == ubuntu-* ]]; then
+        if [[ "$SPREAD_SYSTEM" == ubuntu-* ]]; then
+            apt install -y pollinate
```
Comment (chipaca) [inline tests/lib/prepare.sh]: could you add a -q to that, and/or a `quiet` at the beginning? Without either of those, we'll have progress bars in the travis logs

File: tests/lib/prepare.sh [hunk 2]
```diff
@@ -164,6 +164,11 @@ EOF
     fi
 
     if [[ "$SPREAD_SYSTEM" == debian-* || "$SPREAD_SYSTEM" == ubuntu-* ]]; then
+        if [[ "$SPREAD_SYSTEM" == ubuntu-* ]]; then
```
Comment (chipaca) [inline tests/lib/prepare.sh]: actually pollinate is "just" a single-file bash script; we could ship it as part of the build to places where it doesn't already exist. Something for another PR though.

**Comment by morphis** [comment]
Nice! LGTM

File: tests/lib/prepare.sh [hunk 3]
```diff
@@ -164,6 +164,11 @@ EOF
...
+            apt install -y pollinate
+            pollinate
+        fi
+
         # Improve entropy for the whole system quite a lot to get fast
         # key generation during our test cycles
         apt-get install rng-tools
```
Comment (chipaca) [inline tests/lib/prepare.sh]: while you're here, can you add -q -y (or -qq) to this?

File: tests/lib/prepare.sh (hunk 1 above)
Comment (fgimenez) [inline tests/lib/prepare.sh]: sure, done

File: tests/lib/prepare.sh (hunk 3 above)
Comment (fgimenez) [inline tests/lib/prepare.sh]: done

**Comment by fgimenez** [comment]
@chipaca thanks! we will need it in this case especially :)

**Pull request merged**